setup.py

```diff
@@ -192,11 +192,13 @@ def get_extensions():
 
 def get_requirements():
     return [
+        'torch>=1.2, <1.5',
```

Comment:
`torch` already gets used before program execution gets here, so it's moot to have this check in `get_requirements()`

Reply:
It might be better off to have a `torch` version check somewhere around the top of this file, and then throw a more descriptive error message?

Reply:
I remove as dependencies and extended the condition for warning messages, but building Kaolin generate a lot logs, I'm worried that user might not see it easily.